services/core-web/common/components/tailings/AssociatedDams.tsx

```diff
@@ -12,22 +12,25 @@ import { getTsf } from "@mds/common/redux/reducers/tailingsReducer";
 import moment from "moment";
 import { storeDam } from "@mds/common/redux/actions/damActions";
 import { useHistory } from "react-router-dom";
-import { EditIcon } from "@/assets/icons";
 import { ADD_DAM, EDIT_DAM } from "@/constants/routes";
-import { IDam, INoticeOfDeparture, ITailingsStorageFacility } from "@mds/common";
+import { IDam, ITailingsStorageFacility } from "@mds/common";
 import { RootState } from "@/App";
 import { ColumnsType } from "antd/lib/table";
 import CoreTable from "@/components/common/CoreTable";
+import { EDIT_OUTLINE } from "@mds/common/constants/assets";
+import { EyeOutlined } from "@ant-design/icons";
```

Comment:
Smallish thing, but a big part of what sped up Minespace from Simen's work was actually importing antD icons individually so it doesn't grab everything at once.
```suggestion
import EyeOutlined from "@ant-design/icons/EyeOutlined";
```

Reply:
Thanks, this should be resolved now